src/blocks-transactions-loader/blocks-transactions-loader.ts

```diff
@@ -132,6 +132,16 @@ export class BlocksTransactionsLoader {
           timeout: this.config.get<number>('BLOCK_LOADER_TRANSACTION_TIMEOUT'),
         },
       );
+
+      for (const transaction of block.transactions) {
+        await this.graphileWorkerService.addJob<LoadDescriptionsOptions>(
```

Comment:
I wonder if this would be better to queue up on a per block basis. If we have 80 transactions per block we will get 80 jobs per minute, most of that time will be spent handling spin up/down of graphile workers, no? I am not sure it will be an issue and I won't block on this, but just want to be cautious

Reply:
I don't think there will be that much overhead since the worker is constantly running and just polling for jobs via the queue. I will monitor and adjust if this gets too slow though

Reply:
Yea we ran into something similar with the multiasset phase 3 changes, trying to recall exactly what it was. The issue may have been fixed by the `graphile` upgrade Jason did here:
https://github.com/iron-fish/ironfish-api/pull/1098